easybuild/easyconfigs/s/scikit-image/scikit-image-0.15.0-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'PythonBundle'
+
+name = 'scikit-image'
+version = '0.15.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://scikit-learn.org/stable/index.html'
+description = """Scikit-learn integrates machine learning algorithms in the tightly-knit scientific Python world,
+building upon numpy, scipy, and matplotlib. As a machine-learning module,
+it provides versatile tools for data mining and analysis in any field of science and engineering.
+It strives to be simple and efficient, accessible to everybody, and reusable in various contexts."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+dependencies = [
+    ('Python', '3.7.2'),
+    ('matplotlib', '3.0.3', versionsuffix),
+    ('Pillow', '6.0.0'),
+    ('dask', '2.3.0', versionsuffix),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('networkx', '2.3', {
+        'source_tmpl': '%(name)s-%(version)s.zip',
+        'checksums': ['8311ddef63cf5c5c5e7c1d0212dd141d9a1fe3f474915281b73597ed5f1d4e3d'],
+    }),
+    ('PyWavelets', '1.0.3', {
+        'modulename': 'pywt',
+        'checksums': ['a12c7a6258c0015d2c75d88b87393ee015494551f049009e8b63eafed2d78efc'],
+    }),
+    ('imageio', '2.5.0', {
+        'checksums': ['42e65aadfc3d57a1043615c92bdf6319b67589e49a0aae2b985b82144aceacad'],
+    }),
+    ('imread', '0.7.1', {
+        'checksums': ['c343b546ab54acdb50dbe69d6793d64ead3ba6d585022f5a7ad4687b7f3db79b'],
+    }),
+    (name, version, {
+        'modulename': 'skimage',
+        'checksums': ['df111e654b47e5ea456c50553debe4c5ddd97258894c7ad3b7f2f9f10798e053'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/'],
```

Comment:
This is the default with `PythonBundle`, so `sanity_check_paths` can be removed

Reply:
done